shared/cfg/guest-os/Linux/RHEL/7.6.cfg

```diff
@@ -2,7 +2,7 @@
     no setup
     image_name = images/rhel76
     os_variant = rhel7
-    unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install, with_installation:
+    unattended_install, check_block_size, svirt_install, with_installation:
```

Comment:
Same to https://github.com/avocado-framework/avocado-vt/pull/2224#discussion_r318374259